ambry-router/src/main/java/com.github.ambry.router/RouterUtils.java

```diff
@@ -42,7 +42,7 @@ static BlobId getBlobIdFromString(String blobIdString, ClusterMap clusterMap)
       blobId = new BlobId(blobIdString, clusterMap);
       logger.trace("BlobId created " + blobId + " with partition " + blobId.getPartition());
     } catch (Exception e) {
-      logger.error("Caller passed in invalid BlobId " + blobIdString);
+      logger.trace("Caller passed in invalid BlobId " + blobIdString);
```

Comment:
I am ok with this. Are you going to update the pr for moving the exceptions out from Operation-specific metrics?